R/uk.R

```diff
@@ -25,27 +28,82 @@ get_uk_regional_cases_only_level_1 <- function() {
   # Add or rename standardised variables 
   data <- dplyr::bind_rows(data_list$nation, data_list$region) %>%
     dplyr::mutate(date = lubridate::ymd(date),
-                  # Cases and deaths are by publish date for Scotland, Wales, NI; 
-                  #   but by specimen date and date of death for England
-                  cases_new = ifelse(stringr::str_detect(areaCode, "^E"), 
-                                      newCasesBySpecimenDate,
-                                      newCasesByPublishDate),
-                  cases_total = ifelse(stringr::str_detect(areaCode, "^E"), 
-                                      cumCasesBySpecimenDate,
-                                      cumCasesByPublishDate),
-                  deaths_new = ifelse(stringr::str_detect(areaCode, "^E"), 
-                                      newDeaths28DaysByDeathDate,
-                                      newDeaths28DaysByPublishDate),
-                  deaths_total = ifelse(stringr::str_detect(areaCode, "^E"), 
-                                        cumDeaths28DaysByDeathDate,
-                                        cumDeaths28DaysByPublishDate)) %>%
-    # Hospitalisations and tested variables are consistent across nations
+                  # Cases and deaths by specimen date and date of death 
+                  #   for all nations + regions
+                  cases_new = newCasesBySpecimenDate,
+                  cases_total = cumCasesBySpecimenDate,
+                  deaths_new = newDeaths28DaysByDeathDate,
+                  deaths_total = cumDeaths28DaysByDeathDate) %>%
+    # Hospitalisations and tested variables are only available for nations (not regions)
```

Comment:
perhaps make this a message printed to the console so that users understand that?